doc/btp_ascs.txt

```diff
@@ -241,3 +241,26 @@ Events:
                                         State (1 octet)
 
                 This event indicates that ASE state has been changed.
+
+        Opcode 0x83 - CIS connected
+
+                Controller Index:       <controller id>
+                Event parameters:       Address_Type (1 octet)
+                                        Address (6 octets)
+                                        ASE_ID (1 octet)
+                                        CIS_ID (1 octet)
```

Comment:
I wonder if maybe we should start adding status fields to connected events to allow IUT to fail immediately instead of timing out on missing event in case of error?

(not going to force this since afair none of our events has this, just something to think about)